### DISCUSSION TITLE
About Achievements

### DISCUSSION THREAD

**Discussion by parimalongithub** [community/community#36775] [Other Feature Feedback, Questions, & Ideas] [answered]
I recently got pull shark badge and it was visble on my profile but somwhow it just got disappear from my profile( ps - I aldrady checked the show achievement box in setting ) How can i show the badage on my profile ?

**Comment by matajeu** [comment, marked as answer]
😀
Settings -> Public profile -> Profile settings -> check "Show Achievements on my profile"

**Reply by parimalongithub** [reply]
i tired that already but it is still not showing .

**Reply by matajeu** [reply]
Well... is this about optional individual Achievements setting? Try to go here.

https://github.com/parimalongithub?tab=achievements

[I referred to this place.](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements)

**Reply by matajeu** [reply]
<img width="1175" alt="스크린샷 2022-10-20 오후 11 21 14" src="https://user-images.githubusercontent.com/116011087/196975836-f1647de5-692c-48c7-96a3-a52983830892.png">

**Reply by parimalongithub** [reply]
thanks man you saved my life !

**Reply by matajeu** [reply]
You're welcome. Hope you enjoy life 😊

**Comment by felipsbreno** [comment]
@parimalongithub This occurs with me too.

Which solution you used ?

**Comment by parimalongithub** [comment]
Just follow the thread brother your issue will get resolved!
